bai2/__init__.py

```diff
@@ -1,3 +1,3 @@
-VERSION = (0, 10, 0)
+VERSION = (0, 10, 1)
```

Comment:
In my opinion, this is a bit more significant than a patch-level change. I’d move to 0.11.0.

Reply:
Would you please add a one-liner to the history section of the read-me?

Reply:
fixed